tests/Model/ValueObject/Taxonomy/Label/LabelNameTest.php

```diff
@@ -67,6 +67,22 @@ public function labelNameDataProvider(): array
                 '--',
                 true,
             ],
+            [
+                PHP_EOL,
```

Comment:
I wonder if this should not just be `\r\n` or `\n` as a string because `PHP_EOL` depends on the OS.